railties/lib/rails/generators/testing/assertions.rb

```diff
@@ -23,7 +23,7 @@ module Assertions
         #     end
         #   end
         def assert_file(relative, *contents)
-          absolute = File.expand_path(relative, destination_root).shellescape
+          absolute = File.expand_path(relative, destination_root).gsub(" ", "\\ ")
```

Comment:
@senny Is to avoid breaking this [test](https://github.com/rails/rails/pull/12606/files#diff-1c12e9f760307fdabb690b66537de420R370) in #12606 . It breaks because the [path](https://github.com/rails/rails/pull/12606/files#diff-1c12e9f760307fdabb690b66537de420R371) got `shellescaped`. If `shellescape` is removed from this line, the test pass. 

May I edit the test?